integrationTests/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "cypress_tests",
+  "version": "1.0.0",
+  "description": "Cypres tests for the EAPD tool",
+  "main": "index.js",
+  "scripts": {
+    "cy:open": "./cypress-setup.sh open",
+    "cy:run": "./cypress-setup.sh run",
+    "cy:run:ci": "cypress run"
+  },
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
```

Comment:
I think all of these could be devDependencies, right?

Reply:
I guess.  It won't ever really be built so I am not sure if it will make much of a difference.  I will move them all.